Bindings/Java/Matlab/Dynamic_Walker_Example/DesignMainStarter_Manager.m

```diff
@@ -0,0 +1,75 @@
+function J = DesignMainStarter_Manager(states)
+% This script opens a Model, edits its initial state, runs a forward 
+% simulation.
+
+display(states)
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+model = Model('../Model/WalkerModel.osim');
+
+% Initialize the system and get the initial state
+s = model.initSystem();
+
+% Set the initial states of the model
+editableCoordSet = model.updCoordinateSet();
```

Comment:
`setValue()` is a const method, so AFAIK you don't need the CoordinateSet to be editable (i.e., you can use `getCoordinateSet()` and remove "editable").